app/services/sign_in/session_creator.rb

```diff
@@ -103,10 +106,37 @@ def credential_email
       @credential_email ||= validated_credential.credential_email
     end
 
+    def user
+      @user ||= User.find(user_uuid)
+    end
+
     def user_account
       @user_account ||= user_verification.user_account
     end
 
+    def user_attributes
```

Comment:
Instead of grabbing these from the `User` object, we should store them in the `ValidatedCredential` as a `user_attributes` hash (with the proper checks and everything)

Reply:
Can you explain more on how we should be sourcing the `first_name` & `last_name`, getting them from when they're provided by the CSP in `/callback` to here in `/token`? I'm seeing the `credential_email` is ultimately passed through CodeContainer, should I package that into `user_attributes` and pass the names that way as well?